.github/workflows/frontend-netlify-deploy-pr.yml

```diff
@@ -0,0 +1,140 @@
+# SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+#
+# SPDX-License-Identifier: AGPL-3.0-only
+
+name: Frontend / Deploy PR to Netlify
+
+on:
+  pull_request:
+    branches: [ develop ]
+    types:
+      - labeled
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+
+permissions:
+  contents: read
+  deployments: write
+
+env:
+  NETLIFY_VERSION: 12.0.11
+  NODE_VERSION: 18
+
+defaults:
+  run:
+    working-directory: frontend
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  changes:
+    name: Check for frontend changes
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    outputs:
+      changed: ${{ steps.changed.outputs.files }}
+    steps:
+      - name: Check for frontend file changes
+        uses: dorny/paths-filter@v2
+        id: changed
+        with:
+          filters: |
+            files:
+              - 'frontend/**'
+              - '.github/**'
+              - '.yarn/**'
+
+  deploy:
+    needs: changes
+    if: github.event.pull_request.draft == false || contains( github.event.pull_request.labels.*.name, 'ci:force deployment')
```

Comment:
There's no space in the label name after the colon. Originally we always had that space.